AU2/plugins/util/game.py

```diff
@@ -35,4 +35,14 @@ def soft_escape(string: str) -> str:
     # and remember that code injection without explicit consent is illegal (CMA sxn 2/3)
     if not string.startswith("<!--HTML-->"):
         return escape(string)
-    return string
\ No newline at end of file
+    return string
+
+
+def escape_format_braces(string: str) -> str:
+    """
+    Escapes { and } in a string so that they will be processed correctly by .format
+    This needs to be called on user-input strings passed as part of the message of any inquirer prompt,
+    or as part of the default of inquirer Text prompts (but not the choices or defaults of List of Checkbox prompts)
+    """
+    # need type check because, when escaping default values, `None` is sometimes passed to this function
+    return string.replace("{", "{{").replace("}", "}}") if isinstance(string, str) else string

```

Comment:
why would this not be a string? require inputs to be cast to a `str` or cast them to a str in the function (and take `Any` as the input parameter type)

Reply:
Sometimes `None` is set as the default value of a `LargeTextEntry` component and so without this check trying to sanitise the value throws an error. Casting to `str` doesn't work for that because it gives the string `'None'`.

I could check explicitly for `None` and cast everything else to `str`. Though, it might be better to just change the default value for `LargeTextEntry` to be "" to not have to deal with non-string values at all, which would also mean we wouldn't have to deal with the `None` case when rendering the HTML version of the component.

Reply:
Update: commit [`f0d3afe`](https://github.com/jsyiek/AU2/pull/51/commits/f0d3afe824c0519b1ef55f37f11ef1849b0bbdfe) implements what I talked about above.